ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -172,6 +173,31 @@ patches:
             rbd_store_user=openstack
             rbd_store_pool=images
             store_description=ceph_glance_store
+$(if [[ "${SERVICE_NAME}" == "MANILA" ]]; then
+  cat <<MANILA_EOF
+    - op: add
+      path: /spec/manila/enabled
+      value: true
+    - op: add
+      path: /spec/manila/template/customServiceConfig
+      value: |
+            [DEFAULT]
+            debug = true
```

Comment:
Maybe not add debug here to save space, I will update after this patchset run.

Reply:
makes sense